Examples/MAX78000/CNN/facial_recognition/README.md

```diff
@@ -0,0 +1,208 @@
+# MAX78000 Facial Recognition Demo
+
+
+
+## Overview
+This demo includes **FaceDetection** and **FaceID** CNN models and runs them sequentially on MAX78000 Feather board only. The **FaceID** CNN model weight binary file (Include/weigths_2.bin) needs to be programmed to SD card.
```

Comment:
It's not clear how to program the binary to the SD card.  You may add a sentence to the end:  The **FaceID** CNN model weight binary file (Include/weigths_2.bin) needs to be programmed to SD card using the "SDHC_weights" project which converts the FaceID weights (weights_2.h) into binary and writes it to the SD card.